README.md

```diff
@@ -3,11 +3,36 @@
 Ventana is a client to help connecting and making requests to a SugarCRM REST
 API on multiple versions.
 
-Currently this library provides only a Javascript connector. Please contribute
-to help us grow this client connector.
+Currently this library provides only a Javascript connector.
```

Comment:
Since we're updating this line anyways, perhaps we should link to https://github.com/sugarcrm/rest-php-client too?